web/modules/custom/do_generated_content/src/Plugin/GeneratedContent/NodeCivicthemeAlert.php

```
@@ -0,0 +1,73 @@
+   */
+  protected function bundleValues(int $index): array {
+    $schedule = CaseMatrix::cycle(self::SCHEDULES, $index);
+
+    $values = [
+      'field_c_n_alert_type' => $this->nodeOption('field_c_n_alert_type', $index),
+      'field_c_n_body' => [
+        'value' => $this->helper::staticHtmlParagraph(),
+        'format' => 'civictheme_rich_text',
+      ],
+      'field_c_n_date_range' => [
+        'value' => RelativeDate::format($schedule[0], 'Y-m-d\TH:i:s'),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Drupal\do_generated_content\Generator\RelativeDate' in method 'bundleValues'. (undefined)

(StaticAccess)